apps/condo/pages/ticket/index.tsx

```diff
@@ -361,10 +372,45 @@ const TicketsTableContainer = ({
         skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
     })
 
+    const [loadNewTicketCount] = useLazyQuery(TicketGQL.GET_COUNT_OBJS_QUERY, {
+        onCompleted: ({ meta: { count } }) => {
+            if (!isNull(ticketsCountRef.current) && ticketsCountRef.current !== count) {
+                const totalNewTicketsCount = count - ticketsCountRef.current
+                const iconPath = totalNewTicketsCount > 9
+                    ? '/favicons/infinity.svg'
+                    : `/favicons/${totalNewTicketsCount}.svg`
+                const newTitle = totalNewTicketsCount > 9
+                    ? intl.formatMessage({ id: 'pages.condo.ticket.index.manyNewTicketsTitle' })
```

Comment:
Do we really want to change the tab name to some message instead of the section name? There was a "Ticket" section and it became "Lots of new tickets". Perhaps just an icon with a number or infinity is enough

Reply:
Title and tab icon will periodically change from original tab title and icon to a new one